packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -52,7 +52,26 @@ This package contains documentation for rubygem-%{gem_name}.
 
 %build
 
+%pre
+for i in ansible ansible_galaxy; do
+  if [ -d %{foreman_proxy_dir}/.$i ]; then
+    mv %{foreman_proxy_dir}/.$i %{buildroot}%{_localstatedir}/lib/foreman-proxy/$i
+  else
+    mkdir -p %{buildroot}%{_localstatedir}/lib/foreman-proxy/$i
+  fi
+done
+
+if [-f %{foreman_proxy_dir}/.ansible.cfg ]; then
+  mv %{foreman_proxy_dir}/.ansible.cfg %{buildroot}%{_sysconfdir}/foreman-proxy/ansible.cfg
+else
+  touch %{buildroot}%{_sysconfdir}/foreman-proxy/ansible.cfg
```

Comment:
I think you could leave this out and place an example file in `%install`. I'll need to verify this.